schematools/contrib/django/models.py

```diff
@@ -570,3 +570,17 @@ def related_model(self):
         dataset_name, table_name, column_name = [to_snake_case(part)
                                                  for part in self.relation.split(":")]
         return apps.all_models[dataset_name][table_name]
+
+
+ACTIVE_PROFILES = None
+
+
+def get_active_profiles():
+    if ACTIVE_PROFILES is not None:
+        return ACTIVE_PROFILES
+
```

Comment:
Ik zou hier eerst nog toevoegen 

ACTIVE_PROFILES = [profiles for profile in Profile.objects.all()]
return ACTIVE_PROFILES